packages/cascara/src/ui/Table/fixtures/TableDX.test.js

```diff
@@ -0,0 +1,70 @@
+import { render, screen, within } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import cosmosFixtures, { dataResults } from './TableDX.fixture';
+
+const {
+  Loading: loadingFixture,
+  Empty: emptyFixture,
+  'Data Only': dataOnlyFixture,
+  'Data w/ dataDisplay': dataWithDisplayFixture,
+} = cosmosFixtures;
+const RESULTS_KEYS = Object.keys(dataResults[0]);
```

Comment:
This feels kind of gross. It does make our fixtures show nice in the Cosmos UI... but maybe in the long run, maybe we should just label these fixtures with the same camel case fixture naming in the fixture file so it is easier to just import them directly here.

Reply:
it's ok, this is way better than the tricky titles I chose, i.e. `EdiTable` 😃